Tests/LibCrypto/TestCurves.cpp

```diff
@@ -84,7 +84,8 @@ TEST_CASE(test_secp256r1)
         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
     };
 
-    u8 alice_public_key_data[64] {
+    u8 alice_public_key_data[65] {
+        0x04,
```

Comment:
This commit seems also like a candidate for squashing
(LibCrypto: SECP256r1 accept points in the full format)